### ISSUE TITLE
Add a parameter to avoid double quote in the DDL/DML commands.

### ISSUE THREAD

**Issue by joaoluizcadore** [typeorm/typeorm#9797]
### Feature Description

Typeorm generates the DDL with the name enclosured with double quotes. 
Exemple: ```create table "users" ("id" number, "name" varchar(30))```

The problem is with Oracle database, where a table created with double quotes, has to be used always with double quotes.

### The Solution

My suggestion is to create a parameter on the datasource options, to add or not double quotes in the DDL commands.
If I'm not wrong, in the old versions of TypeORM there was a parameter called ```quoteIdentifiers``` that solves this problem. But it is not available in the latest version.

It was in the deprecated interface ```ConnectionOptions```.

So, if this parameter is false, the DDL should be generated like this:
``` create table user (id number, name varchar2(30)) ```
It it is true:
``` create table "user" ("id" number, "name" varchar2(30))```

### Considered Alternatives

Sorry, I can't think in another solution.

### Additional Context

I said DDL commands, but it should work also with DML.

### Relevant Database Driver(s)

- [ ] aurora-mysql
- [ ] aurora-postgres
- [ ] better-sqlite3
- [ ] cockroachdb
- [ ] cordova
- [ ] expo
- [ ] mongodb
- [ ] mysql
- [ ] nativescript
- [X] oracle
- [ ] postgres
- [ ] react-native
- [ ] sap
- [ ] spanner
- [ ] sqlite
- [ ] sqlite-abstract
- [ ] sqljs
- [ ] sqlserver

### Are you willing to resolve this issue by submitting a Pull Request?

Yes, I have the time, but I don't know how to start. I would need guidance.

**Comment by jmdiazlr-sys** [comment]
I agree with  this requirement, because some database require quoted always if tables where created with quotes, so, if the db is being using by other means, forcing to type quotes in other tools

**Comment by manal617** [comment]
Okay I understand that
